src/services/__tests__/helpers.spec.ts

```diff
@@ -40,21 +40,20 @@ describe('helpers', () => {
       expect(exerciseKey).toBe(0)
     })
 
-    it('should open third exercise of first discipline, if three exercises were finished yet', async () => {
+    it('should open third exercise of first discipline, if two exercises were finished yet', async () => {
       mocked(loadDiscipline).mockReturnValueOnce(Promise.resolve(mockDisciplines(true)[0]))
       mocked(getExerciseProgress).mockReturnValueOnce(Promise.resolve({ '10': { '0': 1, '1': 1, '2': 1 } }))
       const { disciplineId, exerciseKey } = await getNextExerciseWithCheck()
       expect(disciplineId).toBe(mockDisciplines()[0].leafDisciplines![0])
-      expect(exerciseKey).toBe(3)
+      expect(exerciseKey).toBe(1)
```

Comment:
Similar problem as with the test above: In the test data (line 45) 3 exercises are done, but not well enough.
Adjust the test data and change this and the title back to what is was before.